### ISSUE TITLE
Intermittent CRASH in /html/browsers/the-window-object/security-window/window-security.html

### ISSUE THREAD

**Issue by asajeffrey** [servo/servo#15962]
This isn't really a CRASH, it looks like a problem where tinyfiledialogs produces some output, and this confuses the test harness into thinking there's been a crash:
```
  ▶ CRASH [expected OK] /html/browsers/the-window-object/security-window/window-security.html
  │ 
  │ VMware, Inc.
  │ Gallium 0.4 on softpipe
  │ 3.3 (Core Profile) Mesa 12.0.1
  │ ALERT:
  │ 
  │ 
  │  ___________
  │ /           \
  │ | tiny file |
  │ |  dialogs  |
  │ \_____  ____/
  │       \|
  │ missing software! (so we switch to basic console input)
  │ tiny file dialogs on UNIX needs:
  │ 	applescript
  │ or	zenity (version 3 for the color chooser)
  │ or	matedialog
  │ or	kdialog
  │ or	Xdialog
  │ or	python 2 with tkinter
  │ or	dialog (opens a console if needed)
  │ or	whiptail, gdialog, gxmessage or xmessage (really?)
  │ or	it will open a console (if needed) for basic input (you had it comming!)
  │ 
  │ 
  │ Alert!
  │ 
  │ press enter to continue ALERT: RESULT: ["/html/browsers/the-window-object/security-window/window-security.html",0,null,null,[["Window Security testing",0,null,null],["A SecurityError exception must be thrown when window.applicationCache is accessed from a different origin.",1,"assert_true: window.applicationCache should exist. expected true got false",
... lots of output elided ...
  │ 
  └ 
```
cc @jdm @jgraham

**Comment by asajeffrey** [comment]
IRC chat with @nox and @jdm: http://logs.glob.uno/?c=mozilla%23servo&s=15+Mar+2017&e=15+Mar+2017#c631384

TL;DR: this looks like a problem with interleaving output from stderr and stdout, c.f. https://github.com/servo/servo/issues/12797#issuecomment-238865607

**Comment by asajeffrey** [comment]
Fixed by #15967.
